Source/WebKit/WebProcess/cocoa/VideoFullscreenManager.mm

```diff
@@ -206,15 +206,19 @@ static FloatRect inlineVideoFrame(HTMLVideoElement& element)
 
 void VideoFullscreenManager::removeContext(PlaybackSessionContextIdentifier contextId)
 {
-    auto [model, interface] = ensureModelAndInterface(contextId);
-
     m_playbackSessionManager->removeClientForContext(contextId);
 
-    RefPtr<HTMLVideoElement> videoElement = model->videoElement();
+    auto [model, interface] = m_contextMap.get(contextId);
+    ASSERT(model);
+    ASSERT(interface);
+    if (!model || !interface)
+        return;
+
+    Ref videoElement = *model->videoElement();
```

Comment:
This could still crash, potentially. Maybe another ASSERT(model->videoElement()) and early return here?